readers/mocks/messages.go

```diff
@@ -29,7 +29,7 @@ func NewMessageRepository(messages map[string][]mainflux.Message) readers.Messag
 	}
 }
 
-func (repo *messageRepositoryMock) ReadAll(chanID string, offset, limit uint64) []mainflux.Message {
+func (repo *messageRepositoryMock) ReadAll(chanID string, offset, limit uint64, andQuery map[string]string) []mainflux.Message {
```

Comment:
I think that this `and` prefix doesn't improve readability of this code, so maybe we can call this one just `query`.